.github/workflows/validate-plugin-entry.yml

```diff
@@ -50,7 +50,16 @@ jobs:
             console.log(`Repo info: ${owner}/${repo}`);
 
             if (owner.toLowerCase() !== author.toLowerCase()) {
-              addError(`The newly added entry is not at the end, or you are submitting on someone else's behalf. Last plugin in the list is: ${plugin.repo}`);
+              const onBehalfOfOrg = await octokit.rest.orgs
```

Comment:
This will break, on GH actions it would be `github.rest...`, will work the same though.

Reply:
gah sorry forgot to swap it out from my internal testing